pkg/builder/builder.go

```diff
@@ -346,7 +362,7 @@ func (b generalBuilder) checkIfNeedBuild(ctx context.Context) bool {
 		b.logger.Debugf("failed to check manifest update: %s", err)
 	}
 	if !isUpdated {
-		b.logger.Infof("manifest is not updated, skip building")
+		// b.logger.Infof("manifest is not updated, skip building")
```

Comment:
Shall we keep this or not?

Reply:
It is a typo, thanks for the review. I am reverting it.